qt/TrackerView.cc

```diff
@@ -0,0 +1,346 @@
+//
+// Created by nick on 10.11.22.
+//
+
+#include "TrackerView.h"
+
+#include <cstdint> // uint64_t
+#include <map>
+#include <unordered_map>
+#include <utility>
+
+#include <QHBoxLayout>
+#include <QLabel>
+#include <QStandardItemModel>
+
+#include "Application.h"
+#include "FaviconCache.h"
+#include "TrackerViewList.h"
+#include "TrackerViewListDelegate.h"
+#include "Filters.h"
+#include "IconCache.h"
+#include "Prefs.h"
+#include "Torrent.h"
+#include "TorrentFilter.h"
+#include "TorrentModel.h"
+
+enum
+{
+    ACTIVITY_ROLE = TrackerViewList::UserRole,
+    TRACKER_ROLE
+};
+
+TrackerViewList* TrackerView::createActivityList()
```

Comment:
Again, please don't copy-paste code. Instead, refactor the model created by `FilterBar::createActivityCombo()` into a separate class and use that model here.

Same goes for `refreshTrackers()` and other stuff. Code sharing is important.